common/config/.golangci.yml

```diff
@@ -13,6 +13,7 @@ run:
   deadline: 20m
   build-tags:
   - integ
+  - integfuzz
```

Comment:
Similar to above being a common-files change.

Reply:
I'm not sure if this change being merged into common-files will cause failures in the other repos as it automatically applied in the repos. We would need the main_test.go file merged prior to the common-files update.